example/REQUIRE

```diff
@@ -0,0 +1 @@
+ConjugatePriors
```

Comment:
Good to know. The examples IIRC also do some plotting. What will be the stance on plotting? Will it have to be explicitly enabled?
The REQUIRE is in `example` directory, shouldn't it be in the `test` directory?